src/runtime/thread_pool.cc

```diff
@@ -325,9 +339,28 @@ class ThreadPool {
   std::unique_ptr<tvm::runtime::threading::ThreadGroup> threads_;
 };
 
+TVM_REGISTER_GLOBAL("runtime.config_threadpool")
+.set_body([](TVMArgs args, TVMRetValue* rv) {
+    int excl = args[0];
+    bool exclude_worker0 = excl == 1;
+    int mode = args[1];
+    int nthreads = args[2];
+    std::vector<unsigned int> sorted_order;
+    unsigned int num_workers_used = threading::configThreadGroup(mode, nthreads, &sorted_order);
```

Comment:
This can be directly merged into ThreadLocal->UpdateWorkerConfig(mode, threads);
Which then calls thread_group_->Configure(exclude_worker0, mode, nthreads);

Reply:
There is no need to pass sorted list back in to the threadpool